protocol/communities/community.go

```diff
@@ -809,7 +809,7 @@ func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*protobuf.CommunityD
 		return nil, ErrNotAdmin
 	}
 
-	if allowedToSendEvents && o.IsMemberOwnerOrAdmin(pk) {
+	if !isControlNode && o.IsPrivilegedMember(pk) {
```

Comment:
The second part of the condition is not needed. As soon as we are not a control node, we cannot do this I think.

Reply:
this is the fix, this is not changing the behavior when only the control node can manage users. For this, we have a separate ticket and it requires some additional implementation

Reply:
no, because a few lines upper we have a check if it is a control node or a privileged user 
https://github.com/status-im/status-go/pull/3819/files#diff-6d2a8416a631183a7cbd5f7919c6afa15e8576af6be1af1e3f9e5f6f1466516dR805-R810

Reply:
aaah ok, got it. In a way, we could still get rid of the second condition because it's a bit redundant, but whatever.